index.html

```diff
@@ -17,13 +17,14 @@
       <header class="header">
           <h1><a href="#">My Logo</a></h1>
           <nav> 
-            <img src="images/Union.png" alt="hamburgeur menu" class="mobile-img">
-            <!-- <img src="images/desktop-header.svg" alt="hamburgeur menu" class="desktop-img"> -->
+            <a href="#"><img src="images/Union.png" alt="hamburgeur menu" class="mobile-img"></a>
+         
           </nav>
-          <ul class="header-menu">
-            <li>Portoflio</li>
-            <li>About</li>
-            <li>Contact</li>
+          <ul class="header-menu" id="header-menu">
+            <li><a href="#portfolio"  class="link">Portoflio</a></li>
+            <li><a href="#about" class="link">About</a></li>
+            <li><a href="#contact" class="link">Contact</a></li>
+            <li> <a href="#"><img src="images/removenav.svg" alt="Back hamburgeur menu " class="remove mobile-img"></a></li>
```

Comment:
- [ ] as the previous reviewer asked it would be better if you add  the blur effect and try to make the menu cover the whole screen and try to re move the horizontal scroll bar in order to match the Figma design and go to the next project 
![image](https://user-images.githubusercontent.com/61790587/174864820-4be1491c-9ce2-4d74-b19d-e17ce2881d28.png)
![image](https://user-images.githubusercontent.com/61790587/174864830-54bfbf23-5623-437e-aa0c-80cc3d98ca2c.png)
![image](https://user-images.githubusercontent.com/61790587/174864842-7a61e6f1-99b0-4f6b-82c5-bff3bd07e71f.png)